pkg/test/testenv.go

```diff
@@ -45,7 +45,7 @@ import (
 const (
 	clustersToCreate          = 3
 	clusterProtoName          = "cluster-%d-%s"
-	cassOperatorVersion       = "v1.17.2"
+	cassOperatorVersion       = "master"
```

Comment:
This probably won't work correctly..

Reply:
It does, it's used to pull the CRDs. I can switch back to v1.17.2 though since we have no CRD changes that we'd leverage here.